app/src/main/java/com/example/quizacademytask/StackListFragment.kt

```diff
@@ -30,33 +29,37 @@ import kotlinx.coroutines.runBlocking
 import okhttp3.*
 import java.io.IOException
 
-private val courseId: Long = 28
-private lateinit var swipeContainer: SwipeRefreshLayout
-private lateinit var recyclerView: RecyclerView
-private lateinit var stacksAdapter: SimpleAdapter
-private lateinit var stacksList: ArrayList<String>
-private lateinit var stackMap: HashMap<String, CardStack> //Pairing stack names and the stacks
-private lateinit var courseJSON: String
-private lateinit var idlingResource: CountingIdlingResource
-private lateinit var courseDAO: CourseDAO
-private lateinit var cardStackDAO: CardStackDAO
-private lateinit var cardDAO: CardDAO
-private lateinit var courseObj: CourseDTO
-private lateinit var appContext: Context
-private lateinit var binding: FragmentStackListBinding
-var isTablet: Boolean = false
 
 class StackListFragment : Fragment(), SimpleAdapter.OnItemClickListener {
 
+    private val courseId: Long = 28
```

Comment:
this is a constant and it would be better to have it upper cased and defined in companion object 
```kotlin
companion object {
  private const val COURSE_ID = 28L
}
```